components/brave_ads/core/internal/catalog/catalog_url_request_json_reader.cc

```diff
@@ -31,7 +31,7 @@ absl::optional<CatalogInfo> ReadCatalog(const std::string& json) {
           data::resource::kCatalogJsonSchemaFilename);
 
   if (!helper::json::Validate(&document, json_schema)) {
-    BLOG(1, helper::json::GetLastError(&document));
+    BLOG(0, helper::json::GetLastError(&document));
```

Comment:
Ideally we should leave this as 1 so that we do not spam the logs, your thoughts?